playbooks/openshift-master/private/config.yml

```diff
@@ -59,7 +59,7 @@
   - openshift_facts:
       role: master
       local_facts:
-        api_port: "{{ openshift_master_api_port | default(None) }}"
+        api_port: "{{ openshift_master_api_port }}"
```

Comment:
I left this in place for openshift_facts because this data gets married up with some urls in openshift_facts and it's too tightly intertwined to remove it at this point.